src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 200px;
+  height: 408px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__block-img {
+  padding: 32px 19px 0;
+}
+
+.product-card__name {
+  padding: 36px 16px 4px;
+}
+
+.title {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card__product-code {
+  padding: 0 16px 16px;
+}
+
+.text {
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__block-stars {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 16px 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.reviews {
+  display: flex;
+  align-self: center;
+}
+
+.review {
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__block-price {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 16px;
+}
+
+.price {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: #616070;
+}
+
+.value {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-card__block-button {
+  padding: 16px;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  background-color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
Not fixed

Reply:
Not fixed

Reply:
Fix this also